hangups/conversation.py

```diff
@@ -473,6 +473,55 @@ async def send_message(self, segments, image_file=None, image_id=None,
                 logger.warning('Failed to send message: {}'.format(e))
                 raise
 
+    async def add_user(self, *user_ids):
```

Comment:
I think the argument should be a `hangups.user.UserID` for consistency with this class' existing `get_user` method.

I'd also make `user_ids` a list instead of using `*args`.

Reply:
I've made it take a list of user IDs in either form, if that's a reasonable compromise -- typically I've either got the user ID as a string, or a `hangups.user.User` whose `id_` is also a string rather than the wrapper, so it'd be nice not to have the callers doing the wrapping.

Reply:
I see you've added support for taking the various protobuf types. What I was interested in was just `hangups.user.UserID`. I'm fine with supporting strings as well.

Reply:
Ah whoops, hadn't clocked that was distinct from `InviteeID` -- may as well still accept the protobuf classes directly as they're the same structure.